### PR TITLE
Added tests for Avatar

### LINKED ISSUE #986: Add tests for Avatar

### PR DESCRIPTION
**Pull request by prasidhprabudhan**
Fixes #986

### CODE REVIEW

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please take a look

**Comment by ajmaln** [comment]
@amaldinesh7 _a Please review.

File: lib/components/Avatar.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const Avatar = ({
   className = "",
   ...otherProps
 }) => {
-  const [isLoaded, setIsLoaded] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(true);
```
Comment (amaldinesh7) [inline lib/components/Avatar.js:42]: why is this change required?

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@prasidhprabudhan _A Please check the comments

File: lib/components/Avatar.js (hunk 1 above)
Comment (ajmaln) [reply lib/components/Avatar.js:42]: @amaldinesh7 this is to render the image initially and fallback to initials on error loading the image. I have updated the condition `shouldDisplayInitials` as well for this.

Comment (ajmaln) [reply lib/components/Avatar.js]: I have refactored the code for the image loading error state.

**Review by amaldinesh7 — APPROVED** [review]
